worlds/khrecom/__init__.py

```diff
@@ -0,0 +1,175 @@
+from typing import List
+
+from BaseClasses import Tutorial
```

Comment:
```suggestion
from BaseClasses import Tutorial, MultiWorld
```
And then in your world class's init, you can remove the quotes around MultiWorld

Reply:
Added this import